src/scales/scale.time.js

```diff
@@ -338,7 +338,7 @@ module.exports = function(Chart) {
 			var me = this;
 			if (!value || !value.isValid) {
 				// not already a moment object
-				value = moment(me.getRightValue(value));
+				value = me.parseTime(me.getRightValue(value));
```

Comment:
`this` would work, but `this` cannot be minified. Since `this` would be used in a bunch of places, we do `var me = this` so that `me` can be minified in the function. In a single function this doesn't save much, but over the entire lib it saves a few kB.